prometheusbp/metrics_buckets_spec.go

```diff
@@ -0,0 +1,28 @@
+//go:build metrics_spec
+// +build metrics_spec
```

Comment:
I don't really think we should use a build tag for this.

this means services need to change their build process to opt-in for the things defined in the spec. if we really want to use a build tag it should be flipped: that services need to opt-in to keep their dashboards from being broken (if they actually hardcoded the current buckets in their dashboards).

but really what this potentially breaks are dashboards using the old buckets for apdex scores. a communication on the release is better than doing all these hassle.

Reply:
I'm on the fence here.  This is definitely a behavior change and it could break people, and so I'm somewhat inclined to not make it the default, but I could easily be convinced that we should do so, and just try to communicate to folks what is happening so they can set the build tag when they update if they know their dashboards will care.

I think the build tag is the right approach for allowing either opt-in or opt-out, whichever we go with.

Reply:
Hmm, I suppose that if the default is the new behavior and users have to explicitly opt out, then they have to do an explicit action either way (add the build tag or update their dashboards).

So I think my instinct is opt-in > no build tag > opt-out

Reply:
Build tags come with a variety of issues, it's not only updating your build process but also local development, tooling having issues to detect it for linting, etc.